test/data/getParentFP.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
How did you generate this data?

Reply:
`commands` are the data that client sents to ledger and responses that it returns.
This is for future compatibility checks and you can grab them from `ledger.js` after serialization and before deserialization.

Other data, like public key is just from the ledger whatever seed I had that time I logged this information. Now that this is hard coded, this should be deterministic.